### PR TITLE
Send currency to solitude for Boku (bug 1130119)

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: webpay/pay/tests/test_tasks.py [hunk 1]
```diff
@@ -392,7 +392,7 @@ def setUp(self):
                             'icons': {'64': 'http://app/i.png'},
                             'name': 'Virtual Sword',
                             'description': 'A fancy sword'}}}
-        self.prices = {'prices': [{'price': 1, 'currency': 'EUR'}]}
+        self.prices = {'prices': [{'price': '0.89', 'currency': 'EUR'}]}
```
Comment (andymckay) [inline webpay/pay/tests/test_tasks.py]: here you are sending it as str and over here: https://github.com/mozilla/solitude/pull/317/files#diff-c699da5bcbd06b92e39a87e0568a2df5R242 doing a multiplication... we aren't going to multiply a string I take it?

Comment (andymckay) [inline webpay/pay/tests/test_tasks.py]: maybe a Decimal('0.89') would be better?

**Comment by andymckay** [comment]
r+wc

File: webpay/pay/tests/test_tasks.py (hunk 1 above)
Comment (kumar303) [inline webpay/pay/tests/test_tasks.py]: it is posted to solitude as a string but the form field is of type Decimal so it gets converted to a decimal: https://github.com/mozilla/solitude/pull/317/files#diff-7d54df5fa0a0a174145b48623082fb09R83

**Pull request merged**